### DISCUSSION TITLE
To specify SSH port different than default

### DISCUSSION THREAD

**Discussion by ratermir** [pyinfra-dev/pyinfra#1047] [General]
Hello, maybe stupid question, but ... in the inventory definition, there is an array with the target SSH hosts. But how can I specify non-default port of target SSH daemon, i.e. other than 22 (which is SSH default so it needn't be specified) - in the inventory definition ? 

I am pretty new in the pyinfra, it like it a lot, I still explore its options. I really appretiate that everything is clear and stright-forward, also that it doesn't invent anothe new language but uses python instead.

Documentation is quite good, but there are missing such details (or maybe I missed it?)

**Comment by maisim** [comment]
Hi @ratermir,

There are some reserved words in the host data used to configure connectors:

https://docs.pyinfra.com/en/2.x/inventory-data.html#connecting-with-data

Available settings for the ssh connector are here:
https://docs.pyinfra.com/en/2.x/connectors/ssh.html

An other way is to set ssh settings in your .ssh/config 
```
Host incusdev2
  Host 192.168.111.111
  Port 22222
  User root
```
